doc/psidk/src/system-service/admin-sys.md

```diff
@@ -47,3 +48,94 @@ This option controls access to the [HTTP API](../http.md#node-administrator-serv
 - Builtin services only: Allows builtin services to access the admin API. This is the default and should rarely need to be changed.
 - All services: Allows any service to access the admin API. This should only be used for trusted private chains.
 - Disabled: The admin API will not be available. Configuration changes can only be made via the command line and config file, which require a server restart. Disabling the admin API will disconnect the administrator service.
+
+## Monitoring Dashboards
+
+Psinode monitoring is powered by metrics collected and fed into Prometheus. Then, Grafana is used to manage dashboards and visualize all the collected data.
+
+The easiest way to run these services is to use docker containers, since we prepared a docker-compose file with all the needed services.
+
+**All the following instructions are assuming you are working with the files under the Psibase repo, in the `/services/user/AdminSys/monitors` directory.**
+
+### Running with docker
+
+1. Update the `prometheus.yml` file to have the correct target of your psinode instance. Eg: if it's running locally, outside of your docker network, you can leave as is and add a new built-in service in psinode config, then restart it:
+
+```
+service = host.docker.internal:$PREFIX/share/psibase/services/admin-sys
```

Comment:
You are right... Since the admin-sys is instructed to be behind local servers, protected by reverse-proxies and such, should we enable CORS for the /native/admin/perf ? it's the only needed endpoint.

Reply:
It's a simple GET endpoint. CORS doesn't apply.

Reply:
The current restriction is that `Origin = Host` (always enforced) and `Host ∈ native-services` (controlled by `admin`). I'm primarily concerned about attacks against a local node.
- Without the first restriction (or CORS for requests subject to CORS), any random webpage that you navigate to can read the local node.
- Without the second restriction, if you have a local clone of a public chain, any service on the chain would be able to access the admin interface.

Reply:
Got it. Can you lift the restrictions only for the `/native/admin/perf` then? I think aside from spamming this endpoint, there's no harm in doing so, right?

Reply:
@swatanabe just tested it but I forgot about the log.. it did not work because I can't connect to the websockets using the `host.docker.internal` 😢

Reply:
The log endpoint definitely needs to stay restricted. What if I change the second restriction to `Host ∈ native-services` OR `Host` does not end with the root host and also make the native code return 404 for such hosts? That will still block access for any on-chain service, but will allow direct access using other host names. Do you see any problems with that? Do we have any use that requires dispatching arbitrary hosts to wasm?

Reply:
Nope, that isn't safe: if the root host is changed, then pages loaded using the previous hostname gain admin access.